src/rpm/const.go

```diff
@@ -0,0 +1,124 @@
+package rpm
+
+const (
+	defaultFileUser   = "root"
+	defaultFileGroup  = "root"
+	defaultFileLang   = ""
+	defaultFileLinkTo = ""
+	emptyDigest       = ""
+
+	headerSignatures = 62
+	headerImmutable  = 63
+
+	hashAlgoSHA256 = 8
+
+	// XXX
+	fileFlag = 1 << 4
+	dirFlag  = 0
+
+	rpmTypeNull        = 0
```

Comment:
May be iota could/should be used here

Reply:
I don't use iota because this constants aren't defined by me, it defined by RPM format. 
I think it's more convenient to define such constants without using iota.